guide/payments/contacts.md

```diff
@@ -0,0 +1,231 @@
+
+### The basics
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/contacts/contact.png"
+   retina = "/assets/images/guide/payments/contacts/contact@2x.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+On the surface, a contact entry represents a person, business or other entity that the user sends payment to, or receives payments from.
+
+- A name
+- An image (profile, avatar, company logo…)
+- A list of identifiers
+- Options
+- Prior interactions
+- Organizational features like tags & favorites
+
+Typical user flows involving contacts are:
+
+- Choosing a contact to send bitcoin to
+- Assigning a contact to a generated payment request
+- Browsing the wallet payment history
+- Organizing contacts (merging duplicates, deletion, etc)
+
+</div>
+
+Let’s go over each of these.
+
+### Contact names & images
+
+{% include image-gallery.html pages = page.imagesNames %}
+
+Various payment requests may include labels and descriptions that may be interpreted as contact names. Similarly, Lightning nodes may have aliases assigned to serve as more human-friendly identifiers. Since all of these are optional, a contact system cannot fully rely on automatically detecting and organizing contacts for the user.
+
+A second aspect of this is that these names can be easily spoofed, there is no guarantee that they are accurate. So wallets should parse these identifiers and suggest them to users for their approval.
+Identifiers
+When we think of traditional contacts, we think of phone numbers, emails, and physical addresses. These identifiers are typically static and reliable for long periods of time. This is different in bitcoin, as each payment has its own address or invoice with a unique identifier. There are several approaches to re-usable identifiers (payment request formats), each with its own advantages and disadvantages.
+
+##### On-chain identifiers
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/contacts/on-chain-identifiers.png"
+   retina = "/assets/images/guide/payments/contacts/on-chain-identifiers@2x.png"
+   alt-text = ""
+   width = 250
+   height = 116
+   layout = "float-right-desktop -background -shadow"
+%}
+
+There are no good options for re-usable identifiers for on-chain payments, primarily due to privacy concerns.
+
+- An on-chain bitcoin address can technically be reused and receive multiple payments. This is highly discouraged, since it allows participants and observers to trace the activity.
+- Extended public keys allow for the generation of many on-chain addresses. A user could share this key with others so they can generate addresses as needed. This is also discouraged, as anyone with this key can observe all transactions in the wallet.
+
+</div>
+
+##### Lightning identifiers
```

Comment:
As we are going with Lightning first, we should discuss these before on-chain. I'd just combine these two sections into one "Reusable payment requests" section that covers both.